spec/mongoid/traversable_spec.rb

```diff
@@ -251,4 +251,164 @@
       end
     end
   end
+
+  describe "#discriminator_key" do
+
+    context "when the discriminator key is not set on a class" do
+      it "equals _type" do
+        expect(Instrument.discriminator_key).to eq("_type")
+      end
+
+      it "the global discriminator key is _type" do
+        expect(Mongoid.discriminator_key).to eq("_type")
+      end
+
+      it "child discriminator keys equal _type: Piano" do
+        expect(Piano.discriminator_key).to eq("_type")
+      end
+
+      it "child discriminator keys equal _type: Guitar" do
+        expect(Guitar.discriminator_key).to eq("_type")
+      end
+    end
```

Comment:
I don't mean to be a stickler for rspec conventions (lol) but could you change these test descriptions so the line reads like a sentence? (e.g. `it "sets the global discriminator key to _type"`)

Reply:
sure!